web/groups/hlt.json

```
@@ -288,6 +290,11 @@
   "HLTHGCalDigisInRegionsProducer|": "HGCal",
   "HLTHGCalRecHitsInRegionsProducer|hltRechitInRegionsHGCAL": "E/Gamma",
   "HLTHGCalRecHitsInRegionsProducer|": "HGCal",
+  "HGCalSoARecHitsProducer@alpaka|": "HGCal|Alpaka",
+  "HGCalLayerClustersFromSoAProducer|": "HGCal|Alpaka",
```

Shouldn't we put the "FromSoA" ones under "Conversion" ?
```suggestion
  "HGCalLayerClustersFromSoAProducer|": "HGCal|Conversion",
```